keploy.sh

```diff
@@ -134,11 +134,29 @@ installKeploy (){
             # Just update the PATH in .zshrc or .bashrc, no alias needed
             PATH_CMD="export PATH=\"\$HOME/.keploy/bin:\$PATH\""
             if [[ "$current_shell" = "zsh" || "$current_shell" = "-zsh" ]]; then
-                append_to_rc "$PATH_CMD" "$HOME/.zshrc"
+                if [ -f "$HOME/.zshrc" ]; then
+                    if ! grep -q "$PATH_CMD" "$HOME/.zshrc"; then
+                        append_to_rc "$PATH_CMD" "$HOME/.zshrc"
+                    fi
+                else
+                    export PATH="$PATH_CMD"
+                fi
             elif [[ "$current_shell" = "bash" || "$current_shell" = "-bash" ]]; then
-                append_to_rc "$PATH_CMD" "$HOME/.bashrc"
+                if [ -f "$HOME/.bashrc" ]; then
+                    if ! grep -q "$PATH_CMD" "$HOME/.bashrc"; then
+                        append_to_rc "$PATH_CMD" "$HOME/.bashrc" 
+                    fi
+                else
+                    export PATH="$PATH_CMD"
+                fi
             else
-                append_to_rc "$PATH_CMD" "$HOME/.profile"
+                if [ -f "$HOME/.profile" ]; then
+                    if ! grep -q "$PATH_CMD" "$HOME/.profile"; then
+                        append_to_rc "$PATH_CMD" "$HOME/.profile"
+                    fi
+                else
+                    export PATH="$PATH_CMD"
```

Comment:
instead of adding these cases we can create a new function where we would call append_to_rc to append and depending upon  exisstence of  /.bashrc/ or zshrc condition append to rc ..if not existent already

Reply:
@khareyash05 Done , Please have a look